mithril.js

```diff
@@ -248,8 +248,21 @@ var m = (function app(window, undefined) {
 			else if (data.tag === "svg") namespace = "http://www.w3.org/2000/svg";
 			else if (data.tag === "math") namespace = "http://www.w3.org/1998/Math/MathML";
 			if (isNew) {
-				if (data.attrs.is) node = namespace === undefined ? $document.createElement(data.tag, data.attrs.is) : $document.createElementNS(namespace, data.tag, data.attrs.is);
-				else node = namespace === undefined ? $document.createElement(data.tag) : $document.createElementNS(namespace, data.tag);
+				if (m.redraw.strategy() === "existing") {
```

Comment:
I've changed my mind here. Rather than try to figure out why our nodes don't match the dom, we simply throw with a detailed error message. I like this better for now as we can see how common this is and determine how/if we want to fix problems going forward.